files/templates/additional-scrape-configs.template

```diff
@@ -1,15 +1,188 @@
-- job_name: "docker"
+# Additional scrape configuration for the tenant clusters components
+# Add scrape configuration for docker 
+- job_name: docker
```

Comment:
Can we align the job names w/ the job names generated by the prometheus-operator? Being consistent here would be neat

Reply:
Sure :)